client/src/components/Troubleshooting/Troubleshooting.jsx

```diff
@@ -1,42 +1,379 @@
-import React from 'react'
-import NavBar from '../NavBar/NavBar'
-import "./Troubleshooting.css"
+import React from "react";
+import NavBar from "../NavBar/NavBar";
+import "./Troubleshooting.css";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+function Troubleshooting({
+  handleSignInOpen,
+  handleCreateAccOpen,
+  isLoggedIn,
+  draintype,
+}) {
+  const [troubleshooting, setTroubleshooting] = useState({});
+  
+
+  function handleOnTroubleshootingChange(key, val) {
+    let newForm = {
+      draintype: troubleshooting.draintype,
+      issue: troubleshooting.issue,
+      changeInVolume: troubleshooting.changeInVolume,
+      persistentIncrease: troubleshooting.persistentIncrease,
+      persistentincreasechange: troubleshooting.persistentincreasechange,
+      damaged: troubleshooting.damaged,
+      flushing: troubleshooting.flushing,
+      persistentDecrease: troubleshooting.persistentDecrease,
+      drainconnections: troubleshooting.drainconnections,
+      thinnerQuantityChange: troubleshooting.thinnerQuantityChange,
+      changeInQuality: troubleshooting.changeInQuality,
+      bloodThickness: troubleshooting.bloodThickness,
+      thickBloodFainting: troubleshooting.thickBloodFainting,
+    };
+    newForm[key] = val;
+    setTroubleshooting(newForm);
+  }
 
-function Troubleshooting({handleSignInOpen, handleCreateAccOpen, isLoggedIn, troubleshooting, handleOnTroubleshootingChange}) {
   return (
-    <div className='troubleshooting'>
-        <NavBar
+    <div className="troubleshooting">
```

Comment:
truth be told I don't see an ez or quick way to make this faster. Looks like a rough change